meshroom/ui/commands.py

```diff
@@ -196,10 +206,21 @@ def redoImpl(self):
         self.duplicates = [n.name for n in duplicates]
         return duplicates
 
-    def undoImpl(self):
-        # delete all the duplicated nodes
-        for nodeName in self.duplicates:
-            self.graph.removeNode(nodeName)
+
+class DuplicateNodeListCommand(_DuplicateNodes):
```

Comment:
Don't need to have 2 classes to do it with one of multiple nodes and inheritance, etc.
Either we keep `DuplicateNodeListCommand` and always use it (even for a single node) or keep `DuplicateNodeCommand` and group the commands for multiple nodes.